src/SonarScanner.MSBuild.Common/RuntimeInformationWrapper.cs

```diff
@@ -19,12 +19,10 @@
  */
 
 using System.Runtime.InteropServices;
-using SonarScanner.MSBuild.Common.Interfaces;
 
 namespace SonarScanner.MSBuild.Common;
 
-public class RuntimeInformationWrapper : IRuntimeInformationWrapper
+public class RuntimeInformationWrapper
 {
-    public bool IsOS(OSPlatform osPlatform) =>
-        RuntimeInformation.IsOSPlatform(osPlatform);
+    public virtual bool IsWindows => RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
```

Comment:
This removes premature generalization that complicates usages, as we never need to ask any other question than `IsWindows` for now, and foreseeable future.

Once we have that need, it will be easy to update.